microshift.py

```diff
@@ -211,7 +212,7 @@ def iso_builder(h: host.Host, name_of_final_iso: str, version: str) -> None:
 
 def deploy(cluster_name: str, node: NodeConfig, external_port: str, version: str) -> None:
     lh = host.LocalHost()
-    bmc = host.bmc_from_host_name_or_ip(node.node, node.bmc_ip, node.bmc_user, node.bmc_password)
+    bmc = BMC.from_ip(node.bmc_ip, node.bmc_user, node.bmc_password)
```

Comment:
Please test that both regular openshift and microshift are able to be deployed with your changes.